### PR TITLE
Wallet Settings: Tweaked import keypair flows to work around current inability to know which keypairs are fully imported on which synced device

### LINKED ISSUE #11968: Wallet Settings: Tweaked import keypair flows to work around current inability to know which keypairs are fully imported on which synced device
Modifications have been made to the keypair import flows (in the `Wallet` page of the `Settings | Desktop` file). 

**Why?**
The previous iteration of the keypair import flows relied on us being able to know and communicate to the user which devices have which keypairs imported. This presumption in the previous design iteration allowed us to direct the user to a specific device depending on what keypair they wanted to import. If a keypair the user wanted to import was currently imported and fully functional on device B, instructions could be provided on device A to head to device B in order to generate the encrypted keypair QR. Given this information isn't currently known, the flows have been updated and generalised in various ways to avoid reliance on this information.

The previously designed flows can be found in the `Archive` pages of the `Settings | Desktop` file for comparison:
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-606636&mode=design&t=kDsJy0gwUXRt62bH-4

The new designs can be found in their place in the `Wallet` page of the `Settings | Desktop` file:
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-535102&mode=design&t=x4R2VWW5FQXxodXP-4

**Detailed list of changes:**

**1) Notification centre item relating to import of keypair to another of the user's synced devices:**

New copy: `Rainbow keypair was added to one of your synced devices`
Old copy: `Rainbow keypair was added to: [device name]`

Screenshot 2023-08-21 at 16 23 23

--

**2) Orange alert message on non-imported keypair** 

New copy: `Import keypair to use derived accounts`
Old copy: `Added to [device name] - [date imported DD MMM YYYY]`

Screenshot 2023-08-21 at 16 26 06

--

**3) Import missing keypairs dialog**

Previously, this dialog would group missing keypairs by the device they can be imported from. The dialog has been reformatted to avoid referencing devices. Instead of device groupings, the user is presented with 2 groups of options. 
- First, they can 'Import via scanning encrypted QR` - this brings up the QR scanner dialog and the user can scan whichever device they wish (the user is relied on to determine which device they scan based on their knowledge of where the required keypairs are fully imported).
- Secondly, they can select individual keypairs to import via seed phrase / private key (depending on how the keypair was originally imported) or QR

- Additionally, we've removed any of the sub-flows that showed differences in the user's flow depending on if keypair they wanted to import was on just one or multiple devices (we no longer need the ability for the user to select which device to import from in the event the keypair they want to import is imported on multiple of their synced devices).

Screenshot 2023-08-21 at 16 30 19

--

**4) QR dialog**

Given we no longer know which device to direct the user to in order to import specific keypairs, the QR dialog title now omits the device name:

New copy: `Scan encrypted keypair QR code`
Old copy: `Scan encrypted QR from [device name]`

In addition, we've taken the opportunity to make the CTA on how to generate the QR on the other device more descriptive:

New copy: `How to display the QR code on your other device`
Old copy: `How to generate the QR`

(This may also need to be rolled out to other sync/QR flows for consistency)

Screenshot 2023-08-21 at 16 34 46

--

**5) QR dialog - how to generate the QR on the other synced device instructions**

Given we no longer know which device to direct the user to in order to import specific keypairs, the QR dialog instructions on how to generate the QR on the other device now avoids directing the user to specific devices:

New copy: `1. Open Status on the device you want to import from`
Old copy: `Open Status on [mobile/desktop device name]`

Tabs have also been added to allow the user to select whether they want to import via QR from a desktop or a mobile device

Screenshot 2023-08-21 at 16 39 32

--

**6) Keypair import confirmation toasts **

These have been amended to avoid specifying the device name (which is largely redundant anyway considering the user will be mindful of which device they have just imported from via the QR route as they will have that device to hand to generate the QR on and scan from).

Screenshot 2023-08-21 at 16 46 49

--

**7) Importing keypairs via the keypair context menu**

Related to point 3, the previous flows enabled the user to select which device they wanted to scan the QR from if the keypair they want to import was imported on more than one of their synced devices. Again, this option and associated dialog states have been removed.

Screenshot 2023-08-21 at 16 53 11

--

**8) Importing keypairs at the account level (Navigating into an account page in Wallet settings)** 

'Import keypair to use account' : The description directly beneath this title has been modified to remove reference to a specific device:

New copy: `This account was added to one of your synced devices. To begin using this account you will first need import the associated keypair to this device. Click “Import missing keypair” above.`

Old copy: `This account was added to [device name]. To begin using this account you will first need import the associated keypair to this device. Click “Import missing keypair” above.`

https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-539781&mode=design&t=x4R2VWW5FQXxodXP-4

Screenshot 2023-08-21 at 17 01 27

--

**9) Importing keypairs at the account level (Navigating into an account page in Wallet)**

Similar to 8, reference to the device name has been removed in the `Import keypair to use this account` panel at the top of the page:

New copy: `This account was added to one of your synced devices. To begin using this account you will first need import the associated keypair to this device.`

Old copy: `This account was added to [device name]. To begin using this account you will first need import the associated keypair to this device.`

Screenshot 2023-08-21 at 17 05 13

https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-539713&mode=design&t=x4R2VWW5FQXxodXP-4

- The flows here align with those discussed earlier in removing the device name from the dialog copy and removing the option to select which device to import from. The copy has been shortened slightly in addition:

Screenshot 2023-08-21 at 17 10 02

--

**9) Importing keypairs after Waku backup**

A couple of amends to align with above copy/logic and fixes:

Amber warning copy:
New copy: `Restored from backup. Import keypair to use derived accounts.`
Old copy: `Restored from backup. Re-enter private key to use.`

Menu items: removal of `Import keypair from device via encrypted QR`

https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=18442-538750&mode=design&t=x4R2VWW5FQXxodXP-4

Screenshot 2023-08-21 at 17 12 31

--

As a final point, when talking about `associated accounts` in relation to accounts that are derived from a keypair, we are now changing this language to `derived accounts` as this is more accurate and allows us to familiarise users with this language which is used elsewhere in the app (eg choosing a derivation path when creating an account). We should use `derived accounts` anywhere we previously used `associated accounts` eg in the keypair context menu: `Remove keypair and associated accounts` becomes `Remove keypair and derived accounts`. I can create a separate issue for which screens/components this impacts if needed.

@saledjenic @John-44 @xAlisher @mariocnovoa

### PR DESCRIPTION
**Pull request by saledjenic**
Corresponding `status-go` PR:
- https://github.com/status-im/status-go/pull/3947

Closes #11968

Here is a video of the point 1, added notification to activity center:

https://github.com/status-im/status-desktop/assets/86303051/e6e9959f-5a29-4856-92de-1f262860f39c

### CODE REVIEW

**Comment by saledjenic** [comment]
@all reviewers, each commit refers to a single point from the #11968, reviewing a single commit in a time makes it easier for review.

**Review by alaibe — APPROVED** [review]

File: ui/app/mainui/activitycenter/views/ActivityNotificationNewKeypairFromPairedDevice.qml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Style.current.padding
+            Layout.topMargin: 2
+
+            asset {
+                width: 24
+                height: 24
+                name: "wallet"
+                color: Theme.palette.primaryColor1
+                bgWidth: 40
```
Comment (Khushboo-dev-cpp) [inline ui/app/mainui/activitycenter/views/ActivityNotificationNewKeypairFromPairedDevice.qml:42]: bgWidth > width ?

File: ui/imports/shared/controls/ImportKeypairInfo.qml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    radius: Style.current.radius
```
Comment (Khushboo-dev-cpp) [inline ui/imports/shared/controls/ImportKeypairInfo.qml:13]: this after property declarations?

File: ui/imports/shared/controls/ImportKeypairInfo.qml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    border.width: 1
+    border.color: Theme.palette.directColor8
+    color: Theme.palette.transparent
+
+    property string title: qsTr("Import keypair to use this account")
+    property string info: qsTr("This account was added to one of your synced devices. To use this account you will first need import the associated keypair to this device.")
+    property string buttonName: qsTr("Import keypair")
```
Comment (Khushboo-dev-cpp) [inline ui/imports/shared/controls/ImportKeypairInfo.qml:20]: readonly?

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM. Some very small comments

File: ui/app/mainui/activitycenter/views/ActivityNotificationNewKeypairFromPairedDevice.qml (hunk 1 above)
Comment (saledjenic) [reply ui/app/mainui/activitycenter/views/ActivityNotificationNewKeypairFromPairedDevice.qml:42]: @Khushboo-dev-cpp `width` refers to the icon's width and `bgWidth` to the background's width.

File: ui/imports/shared/controls/ImportKeypairInfo.qml (hunk 2 above)
Comment (saledjenic) [reply ui/imports/shared/controls/ImportKeypairInfo.qml:13]: Technically it's the same, but will change this to have custom props first.

File: ui/imports/shared/controls/ImportKeypairInfo.qml (hunk 3 above)
Comment (saledjenic) [reply ui/imports/shared/controls/ImportKeypairInfo.qml:20]: No, they cannot be readonly cause those are exposed properties, with defined default values, that can be changed from outside.

**Comment by saledjenic** [comment]
@Khushboo-dev-cpp thanks for the review.

**Pull request merged**